### ISSUE TITLE
release question

### ISSUE THREAD

**Issue by SaraOgaz** [astropy/ccdproc#548]
I have sort of two questions.  When will 1.3 be released?  And is it "correct" that version 1.2 is not compatible with astropy 2.0?  I get this incompatibility when I try and upgrade to 2.0 through conda, astroscrappy apparently being used by ccdproc:

UnsatisfiableError: The following specifications were found to be in conflict:
  - astropy 2.0*
  - astroscrappy -> numpy 1.11*
Use "conda info <package>" to see the dependencies for each package.

**Comment by MSeifert04** [comment]
I don't think it's actually a problem with `ccdproc` because it's a pure Python package (except `ccdproc` wasn't build for your used python). It seems like `astroscrappy` and `astropy` conflict because of NumPy. What NumPy/Python are you using and which conda channels are enabled for you?

**Comment by SaraOgaz** [comment]
I figured astroscrappy's numpy pin was the root of the problem, but I only need astroscrappy because it seems to be a ccdproc requirement, at least according to my conda...

For this particular issue I need to use the astroconda channel for most things (our STScI channel).  Currently in Python 2.7, numpy is 1.11 (not from astroconda channel, I think it's just from the default Anaconda).

When I try and update numpy  I get the same astroscrappy conflict...


$ conda install numpy=1.13
Fetching package metadata ...........
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - astroscrappy -> numpy 1.11*
  - numpy 1.13*
Use "conda info <package>" to see the dependencies for each package.


Basically, the astroscrappy requirement for CCDPROC, which leads to astroscrappy's numpy pin, is stopping me from updating things. At least, I'm pretty sure that's what's happening.

**Comment by MSeifert04** [comment]
hm, okay I don't know what's in your `STScI` channel, maybe you could run `conda search astroscrappy`? That normally lists the avaiable `astroscrappy` versions. However in the astropy channel there is astroscrappy built against NumPy 1.13 and Python 2.7, 3.5 and 3.6 so it should be available. Maybe the STScI channel needs new astroscrappy builds?!

**Comment by SaraOgaz** [comment]
Huh, it says I'm getting astroscrappy from the astropy channel, let me mess around some more.

astroscrappy              1.0.5               np111py27_0    astropy

**Comment by SaraOgaz** [comment]
Nvm, it was because i wasn't updating numpy in the meantime, now I'm having other problems that are just related to my old astroconda channel stuff.  I'm going to scrap this env and just start a new one.  Will probably close this issue here in  a second.  Thanks for the help @MSeifert04 !

**Comment by MSeifert04** [comment]
No problem. I hope `ccdproc` will still work with astropy 2.0 but we'll release ccdproc 1.3 soon-ish.
